strategy/measuring-impact.md

```diff
@@ -0,0 +1,148 @@
+
+Scientists will access analysed data that has been generated in their area of research interest in order to exploit it using trusted analysis tools.
+
+Scientists will explore the data in the DCP via a user interface in order to find data relevant to their research interest, before using programmatic techniques and tools to perform more in-depth analyses. They expect the DCP to provide a consistent experience across user interfaces and APIs.
+
+In order for it to be easy to use or develop analysis tools, data in the DCP must be easy to use and the experience of discovering and using this data must be the same whether data is accessed via user interfaces or APIs.
+
+#### Indicators for Success
+
+| Specific aim | Item | Indicator |
+| -----------: | ---- | :-------- |
+| 1 | a. | The same scientists return to the DCP to access data as new datasets are published (i.e. the DCP is retaining existing scientists because the data is useable) |
+|   | b. | Analysed data is accessed more commonly than raw data |
+|   | c. | User requests for additions or modifications to the metadata available are relatively rare |
+| 2 | a. | User feedback and UX research indicates a high degree of satisfaction with DCP user interfaces and APIs |
+| 3 | a. | Relative usage of the UI for data searches, and the API for direct data access, show consistent patterns over time, rather than changing significantly depending on which access mode is easier |
+|   | b. | Search analytics show datasets are discovered using a small number of key searches common to a wide number of scientists |
+|   | c. | All data used in the Human Cell Atlas or tissue-specific cellular atlases (e.g. brain atlas) is present in the DCP |
+
+
+### Aim 3
+
+**Perform regular data releases of high value to single cell researchers.**
+
+#### Value Assumptions
+
+The two main types of scientists, when consuming others’ data from the platform, value consistency of data and metadata, in terms of standards, versions and quality control over the speed of access.
+
+Scientists will analyse comprehensive data using available analysis tools as long as this is easy to do, and to be easy, the DCP must provide access to data that is “pre-harmonised” (i.e. all uses the same standards, processing pipelines and QC criteria).
+
+Scientists will analyse all DCP data, including “historical datasets” (that don’t conform to the latest standards and aren’t possible to automatically migrate), but expect the DCP to curate these historical datasets for them and aren’t prepared to do this themselves.
+
+Scientists will infrequently update their analysis methods, so data from prior releases will continue to be used for a long time after it was initially made available.
+
+#### Indicators for Success
+
+| Specific aim | Item | Indicator |
+| -----------: | ---- | :-------- |
+| 1 | a. | Access statistics for DCP data releases are higher than access statistics for the “live” DCP view |
```

Comment:
Releases are successful if people use them and cite them.  I don't think they are successful only if they are used more than, say, the live view.